scikeras/wrappers.py

```diff
@@ -1310,6 +1322,26 @@ def _type_of_target(self, y: np.ndarray) -> str:
             target_type = type_of_target(self.classes_)
         return target_type
 
+    def _compile_model(self, compile_kwargs: Dict[str, Any]) -> None:
+        if compile_kwargs["loss"] == "auto":
+            if len(self.model_.outputs) > 1:
+                raise ValueError(
+                    'Only single-output models are supported with `loss="auto"`'
+                )
+            if self.target_type_ == "binary":
+                compile_kwargs["loss"] = "binary_crossentropy"
+            elif self.target_type_ == "multiclass":
+                if self.model_.outputs[0].shape[1] == 1:
+                    raise ValueError(
+                        "Multi-class targets require the model to have >1 output unit"
```

Comment:
```suggestion
                        f"Multi-class targets require the model to have >1 output unit instead of {self.model_.outputs[0].shape} units"
```